api/app/schemas/weather_models.py

```diff
@@ -47,11 +47,8 @@ class WeatherModelPredictionValues(BaseModel):
     bias_adjusted_relative_humidity: float = None
     wind_speed: float = None
     wind_direction: float = None
+    delta_precipitation: float = None
     total_precipitation: float = None
```

Comment:
Can we please drop total_precipitation from the response? I think it's just confusing things having it - it only makes sense to keep track of total precipitation on the backend.